setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name = 'xpdacq',
-    version = '1.0.2',
```

Comment:
Fair enough. Your git tags look to be in good shape (latest is 0.3) so I would make this 0.4.0 (or something).